user/tlu/misc/aida_tlu/aida_tlu_connection.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <connections>
     <connection id="aida_tlu.controlhub" uri="chtcp-2.0://localhost:10203?target=192.168.200.30:50001"
-        address_table="file://./aida_tlu_address-fw_version_14.xml" />
+        address_table="file://./aida_tlu_address-fw_version_25.xml" />
```

Comment:
This is not the default firmware yet (mainly a ping to @herkerta ). Also an important message in the upcoming tag